packages/manager/apps/pci-savings-plan/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -3,7 +3,7 @@ import { Params, useParams } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { OsdsBreadcrumb } from '@ovhcloud/ods-components/react';
 import { useNavigation } from '@ovh-ux/manager-react-shell-client';
-import usePciProject from '@/hooks/usePciProject';
+import usePciProject from '../../hooks/usePciProject';
```

Comment:
```suggestion
import usePciProject from '@/hooks/usePciProject';
```
I think this change isn't need